examples/linear/seeds/linear_default.json

```
@@ -137,6 +131,408 @@
+      "isAssignable": true,
+      "isMe": false,
+      "isMentionable": true,
+      "lastSeen": "2025-01-01T00:00:00",
+      "statusEmoji": null,
+      "statusLabel": null,
+      "statusUntilAt": null,
+      "timezone": "Europe/Rome",
+      "url": "https://linear.app/test-org/user/sofia-rossi"
+    }
+  ],
+  "team_memberships": [
```

Biome (2.1.2)

[error] The key team_memberships was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)